owncloudApp/build.gradle

```diff
@@ -47,7 +47,7 @@ dependencies {
     implementation "androidx.room:room-runtime:$roomVersion"
 
     // Koin dependency injector
-    implementation "io.insert-koin:koin-android:3.0.2"
+    implementation "io.insert-koin:koin-android:$koin"
```

Comment:
What's the point for a variable for a usage of just _**one**_ place ?
It only prevents from usage of Intent action in Android Studio

Reply:
<img width="462" alt="image" src="https://user-images.githubusercontent.com/3314607/132813812-07d215c6-62c3-48a1-a3e0-d61755d32155.png">

A variable does not work here

Reply:
We are going to use it in two places again, we restore the variable.